pubspec.yaml

```diff
@@ -10,7 +10,7 @@ environment:
 
 dependencies:
   meta: ^1.6.0
-  http: ^0.13.0
+  http: ^1.0.0
```

Comment:
Why not make it less restrictive, i.e.
```yaml
http: ">=0.13.0 <2.0.0"
```

Reply:
I'm not sure you understood the version restriction above.

Btw, you can always resort to `dependency_overrides` if your requirement for `^1.0.0` is so dire.